### PR TITLE
New title AWS compute services & improved wording

### PR DESCRIPTION
**Pull request by aleon1220**
i moved the new title AWS Compute with the Spinnaker AWS Cloud Provider to the top
i added a link to an official AWS lambda and spinnaker
this sets the stage for new Spinnaker docs on AWS Lambda
the concepts block is at the bottom.
In a future PR i will improve the wording of concepts as it requires some more detail

### CODE REVIEW

**Comment by aimeeu** [comment]
See Slack [thread](https://spinnakerteam.slack.com/archives/CMPS49682/p1595197821280500) for details

File: setup/install/providers/aws/index.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ sidebar:
 
 {% include toc %}
 
+### AWS Compute with the Spinnaker AWS Cloud Provider
+The AWS Cloud Provider allows Spinnaker to release artifacts in some of the [AWS compute services](https://aws.amazon.com/products/compute/) with Spinnaker.
```
Comment (dorbin) [inline setup/install/providers/aws/index.md]: "allows Spinnaker" and "with Spinnaker" seem redundant.

File: setup/install/providers/aws/index.md [hunk 2]
```diff
@@ -7,6 +7,16 @@ sidebar:
 
 {% include toc %}
 
+### AWS Compute with the Spinnaker AWS Cloud Provider
```
Comment (dorbin) [inline setup/install/providers/aws/index.md]: I think it's a good idea to omit the heading at the beginning of the article (though we don't follow that consistently in this doc set). Is this heading necessary, given that the instructions are meant to be just how to set up an AWS provider?

Comment (dorbin) [reply setup/install/providers/aws/index.md]: If you decide this heading is worth keeping, promote it to heading level 2.

File: setup/install/providers/aws/index.md [hunk 3]
```diff
@@ -7,6 +7,16 @@ sidebar:
...
+The AWS Cloud Provider allows Spinnaker to release artifacts in some of the [AWS compute services](https://aws.amazon.com/products/compute/) with Spinnaker.
+
+There are several ways to configure the Amazon Web Services (AWS) Cloud Provider. Choose one or more based on your requirements:
+
+* [Amazon Elastic Compute Cloud (EC2)](/setup/install/providers/aws/aws-ec2/) - - Use this option, if you want to manage [AWS EC2](https://aws.amazon.com/ec2/) via Spinnaker
+* [Amazon Elastic Container Service (ECS)](/setup/install/providers/aws/aws-ecs/) - - Use this option, if you want to manage containers in [AWS ECS](https://aws.amazon.com/ecs/)
+* [Amazon Elastic Kubernetes Service (EKS)](/setup/install/providers/kubernetes-v2/aws-eks/) - Use this option, if you want to manage containers in [AWS EKS](https://aws.amazon.com/eks/). This option uses [Kubernetes V2 (manifest based) Clouddriver](/setup/install/providers/kubernetes-v2)
+* [Amazon Lambda (Lambda)](https://aws.amazon.com/blogs/opensource/how-to-integrate-aws-lambda-with-spinnaker/) - Use this option, if you want to enable [AWS Lambda](https://aws.amazon.com/lambda/) support 
+
 ### Concepts
```
Comment (dorbin) [inline setup/install/providers/aws/index.md]: Beyond the scope of your changes, so feel free to ignore, but...

This was already a level-3 heading before you got here, @aleon1220 , but I'm not sure why. Can you promote to h2?

Also, "concepts" is pretty vague, considering this section is recognizably about AWS accounts. Maybe change it to "About AWS accounts"?

**Review by dorbin — COMMENTED** [review]
Thanks for the PR, @aleon1220 ! Just a few comments.

File: setup/install/providers/aws/index.md (hunk 1 above)
Comment (aleon1220) [reply setup/install/providers/aws/index.md]: i will chage the wording there.

File: setup/install/providers/aws/index.md (hunk 2 above)
Comment (aleon1220) [reply setup/install/providers/aws/index.md]: mmm yes maybe is best to remove that heading. Gonna do it then from my PR.

File: setup/install/providers/aws/index.md (hunk 3 above)
Comment (aleon1220) [reply setup/install/providers/aws/index.md]: OK. I will promote it to h2.
About the concepts, i agree with you is vague. That section relates to how Spinnaker uses AWS IAM to have full permissions to deploy and manage apps in AWS. I could think of a title involving permissions or Identity Access Management.That for a future PR.
Thanks

**Review by aleon1220 — COMMENTED** [review]
@dorbin I have added the improvements and a line to explain the AWS Accounts section.

**Review by dorbin — APPROVED** [review]

**Pull request merged**